### PR TITLE
median_filter.py added

### PR DESCRIPTION
**Pull request by codezoned**
**Fixes issue:**

1st addition to #3511 . More to be added

**Changes:**

### CODE REVIEW

File: code/filters/median_filter.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+    ##3 defines the no of layers for the filter
+    ##It also defines that the filter will be 3x3 size. Change 3 to 5 for a 5x5 filter
+
+    removed_noise = median_filter(arr, 3)
+    img = Image.fromarray(removed_noise)
+    img.show()
+
+
+main()
```
Comment (arnavb) [inline code/filters/median_filter.py]: Adthe if name equals main condition.

Comment (codezoned) [reply code/filters/median_filter.py]: Alright

**Comment by codezoned** [comment]
@AdiChat Can you look into merging this

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

positive_opengenus_first

**Pull request merged**